engine/src/test/java/com/redhat/cloud/notifications/templates/TestRbacTemplate.java

```diff
@@ -115,6 +116,9 @@ private void testTitle(String eventType, String result) {
             case REQUEST_ACCESS:
                 assertEquals("Instant notification - Request access - User Access - Console", result);
                 break;
+            case TAM_ACCESS_REQUESTED:
```

Comment:
Could you please also consider to add a test case on `testBody` method ?

Reply:
Sure, just added